contracts/flashloan/Instapool v2/helpers.sol

```diff
@@ -0,0 +1,115 @@
+pragma solidity ^0.7.0;
+
+import { 
+    IndexInterface,
+    ListInterface,
+    TokenInterface,
+    IAaveLending
+} from "./interfaces.sol";
+
+import { DSMath } from "../../common/math.sol";
+
+import {Variables} from "./variables.sol";
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
```

Comment:
@KaymasJain since this is a upgradable contract. I think we need to create upgradable ownable contract or use OZ upgradable contract.

Reply:
https://docs.openzeppelin.com/contracts/4.x/upgradeable
https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/master/contracts/access/OwnableUpgradeable.sol